examples/bt_move_to.cpp

```diff
@@ -0,0 +1,141 @@
+                                   goal="panda_joint7:-1.57079632679,panda_joint6:3.14"
+                                   stage="{stage_move_to_joint}" />
+       <MoveMTCStageToContainer  container="{container}" stage="{stage_move_to_joint}" />
+       <!-- cartesian pose motion -->
+       <GeometryMsgsPoseStamped  frame_id="panda_link8" position="0,0,0" quaternion="1,0,0,0" pose_stamped="{ik_frame}"/>
+       <GeometryMsgsPoseStamped  frame_id="panda_link8" position="0,0,0.05" quaternion="0,0,0,0" pose_stamped="{goal_pose}"/>
+       <CreateMTCMoveToPose        name="move to -> cartesian pose motion"
+                                   group="panda_arm"
+                                   solver="{rrt_connect}"
+                                   ik_frame="{ik_frame}"
+                                   goal="{goal_pose}"
+                                   stage="{stage_move_to_pose}" />
+      <MoveMTCStageToContainer  container="{container}" stage="{stage_move_to_pose}" />
+      <!-- cartesian point motion -->
+       <GeometryMsgsPointStamped  frame_id="panda_link8" point="0,0.05,0" point_stamped="{goal_point}"/>
+       <CreateMTCMoveToPoint       name="move to -> cartesian point motion"
+                                   group="panda_arm"
+                                   solver="{rrt_connect}"
+                                   ik_frame="{ik_frame}"
+                                   goal="{goal_point}"
+                                   stage="{stage_move_to_point}" />
+      <MoveMTCStageToContainer  container="{container}" stage="{stage_move_to_point}" />
+       <PlanMTCTask              task="{mtc_task}" max_solutions="5" />
+     </Sequence>
+   </BehaviorTree>
+
+ </root>
+ )";
+// clang-format on
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "test_behavior_tree");
+  ros::AsyncSpinner spinner(1);
+  spinner.start();
+
+  BehaviorTreeFactory factory;
+
+  factory.registerNodeType<InitializeMTCTask>("InitializeMTCTask");
+  factory.registerNodeType<CreateMTCPipelinePlanner>("CreateMTCPipelinePlanner");
+  factory.registerNodeType<CreateMTCCurrentState>("CreateMTCCurrentState");
+  factory.registerNodeType<CreateMTCMoveToNamedJointPose>("CreateMTCMoveToNamedJointPose");
+  factory.registerNodeType<CreateMTCMoveToJoint>("CreateMTCMoveToJoint");
+  factory.registerNodeType<CreateMTCMoveToPose>("CreateMTCMoveToPose");
+  factory.registerNodeType<CreateMTCMoveToPoint>("CreateMTCMoveToPoint");
+  factory.registerNodeType<MoveMTCStageToContainer>("MoveMTCStageToContainer");
+  factory.registerNodeType<PlanMTCTask>("PlanMTCTask");
+
+  factory.registerNodeType<GeometryMsgsPoseStamped>("GeometryMsgsPoseStamped");
+  factory.registerNodeType<GeometryMsgsPointStamped>("GeometryMsgsPointStamped");
+  factory.registerNodeType<GeometryMsgsVector3Stamped>("GeometryMsgsVector3Stamped");
+  factory.registerNodeType<GeometryMsgsTwistStamped>("GeometryMsgsTwistStamped");
```

Comment:
Remove lines that register a node that is not used in this demo. E.g. GeometryMsgsTwistStamped.